ui/src/common/components/NewFeatureTag/NewFeatureTag.jsx

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 import { Tag } from 'antd';
+import PropTypes from 'prop-types';
 import styleVariables from '../../../styleVariables';
 
 import './NewFeatureTag.scss';
 
 const GREEN = styleVariables['success-color'];
 
-function NewFeatureTag() {
+function NewFeatureTag({ className }) {
   return (
-    <Tag className="__NewFeatureTag__" color={GREEN}>
+    <Tag className={className} color={GREEN}>
```

Comment:
This is how you add `className` like props:

https://github.com/inspirehep/inspirehep/blob/a59416dab0e592c35b2b38d84412ad0cdd75016d/ui/src/common/components/InlineList/InlineList.jsx#L27

So here would be:

`className={classnames(''__NewFeatureTag__", className)`

and you define the actual class that you're passing, in the parent.